From 83eb0bbdc6777c4f42f6abb958d7613b8381e783 Mon Sep 17 00:00:00 2001
From: Song Zhao <song.zhao@amlogic.com>
Date: Wed, 22 Apr 2020 01:04:26 -0700
Subject: [PATCH] gst-aml-drm: disable parse [1/1]

PD#SWPL-24203

Problem:
Westeros-sink inherits GstBaseSink, which will drop key frames with CSD.

Solution:
Integrate parse function inside westeros-sink, and disable it here.

Verify:
U212 DSTV
---
 gst-aml-drm-plugins-1.0/configure.ac    | 1 -
 gst-aml-drm-plugins-1.0/src/Makefile.am | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/gst-aml-drm-plugins-1.0/configure.ac b/gst-aml-drm-plugins-1.0/configure.ac
index 3a445a1..9847f61 100644
--- a/gst-aml-drm-plugins-1.0/configure.ac
+++ b/gst-aml-drm-plugins-1.0/configure.ac
@@ -86,7 +86,6 @@ AC_SUBST(GST_PLUGIN_LDFLAGS)
 AC_CONFIG_FILES([Makefile
 src/Makefile
 src/secmem/Makefile
-src/secure_parse/Makefile
 src/dummy/Makefile
 ])
 AC_OUTPUT
diff --git a/gst-aml-drm-plugins-1.0/src/Makefile.am b/gst-aml-drm-plugins-1.0/src/Makefile.am
index bfda5e6..562cf53 100644
--- a/gst-aml-drm-plugins-1.0/src/Makefile.am
+++ b/gst-aml-drm-plugins-1.0/src/Makefile.am
@@ -1 +1 @@
-SUBDIRS = secmem secure_parse dummy
+SUBDIRS = secmem dummy
-- 
2.24.1

